src/test/java/org/apache/commons/text/TextStringBuilderTest.java

```diff
@@ -2276,4 +2289,4 @@ public void testWrap_CharArray_Int_Exceptions() {
         assertThrows(IllegalArgumentException.class, () -> TextStringBuilder.wrap(ArrayUtils.EMPTY_CHAR_ARRAY, 1));
     }
 
-}
+}
```

Comment:
@sebx59 
MIssing blank line as the last line (see red icon warning from GitHub above).